packages/story-editor/src/utils/getMediaBaseColor.js

```diff
@@ -93,17 +94,16 @@ export function setOrCreateImage(
   imageData,
   getOnloadCallback = getDefaultOnloadCallback
 ) {
-  const { src, id, height, width } = imageData;
+  const { src, height, width } = imageData;
+  const id = uuidv4();
   return new Promise((resolve, reject) => {
     const NODE_KEY = getImgNodeKey(id);
-    let imgNode = document.body[NODE_KEY];
```

Comment:
One reason for re-using the same image node was not to create a bunch of nodes hanging around in the DOM but just reusing the same image tag. Why is adding a new node every time better than using the same one node?

If I understood correctly, Pascal's question was generally about adding the node to the DOM, this still does it, no?